dnstable_convert.c

```diff
@@ -780,11 +862,11 @@ write_thread(void *clos)
 {
 	struct write_thread_ctx *ctx = (struct write_thread_ctx *) clos;
 	struct mtbl_iter *it = mtbl_sorter_iter(ctx->s);
-	assert(it != NULL);
-
 	const uint8_t *key, *val;
 	size_t len_key, len_val;
 
+	assert(it != NULL);
```

Comment:
It's unlikely to pose a problem, but I'm curious about why this assertion was moved to be after two more variable declarations.

Reply:
Most likely a style / coding convention fix. We generally prefer the style of "declare first, then do things", and fixed this while nearby lines of code were being changed.